android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.bleplx">
+          package="com.bleplx"
+          xmlns:tools="http://schemas.android.com/tools">
+
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
I'm not sure if the library should really add all the permissions in it's own AndroidManifest? Isn't that up to the apps that use it to define?

Reply:
Hi, thank you for this comment.
As mentioned in PR description - we migrated this code (just copied and set correct packages) from external `MultiPlatformBle` library, because it was harder to keep track of changes and issues. 

As for the question. 
We'd have to break compatibility with previous versions if we remove permissions from here. 
Manifest file was already added in that library.